docs/release-notes/v11.5.0.md

```diff
@@ -0,0 +1,217 @@
+---
+id: v11.5.0
+title: Version v11.5.0 Release Notes
+sidebar_label: v11.5.0
+image: "img/release-note-link-preview.png"
+---
+
+_Aug 31st, 2023_
+
+## Console
+
+### New Features
+
+#### Project Settings 
+
+Inside the 'Overview' Area of a Project, you can now access a brand new section called 'Settings', where users with enough administrative permissions can view and manage the settings of their Project. This will speed-up and optimize the overall administration of Projects, which – until now – was only possible exclusively via CMS, making it no longer tedious or error-prone!
+
+![Project Settings](./img/overview/project-settings.png)
+
+For more information about Project Settings, [go to the official documentation](/development_suite/set-up-infrastructure/project-settings.md).
+
+#### Mongo Collections and Views support multiple tags
+
+The collections configured in the section `MongoDB CRUD` and the views configured in the `MongoDB Views` section can have more than one tag. These tags will be then included in the configuration files passed to the `CRUD Service` to have the collections grouped by said tags, in order to simplify the use of the service.
+
+Please note that it is possible to create multiple tags only if the `CRUD Service` configured in the project has version `6.8.0` or superior. In case there are different versions per environment, all the versions defined must be equal or superior to `6.8.0` to configure these tags.
+
+#### Support of Lookup in CRUD Views
+
+In the `MongoDB Views` section, it is possible to enable _lookup_, following specific pipeline requirements to obtain data from _lookup_ collections, simplifying interaction with views and improving support for collections with _one-to-one_ or _many-to-one_ relationships.
+
+More information can be found in the [official documentation](/runtime_suite/crud-service/50_writable_views.md).
+
+Please note that it is possible to enable lookups only if the `CRUD Service` configured in the project has version `6.9.0` or superior. In case there are different versions per environment, all the versions defined must be equal or superior to `6.8.0` to enable lookups.
+
+### Improvements
+
+#### User Profile section 
+
+We have enhanced the viewing and management of user profile settings and preferences, combining them into a single section always accessible from the Console launchbar. By clicking on the new `Profile` option from the dropdown menu, you will now be presented with a section containing two tabs: `Personal info` and `Preferences`.  
+In the `Personal info` tab, you will find a brief summary of your profile and an external link for managing your profile settings on the identity provider.  
+Within the "Preferences" tab, you can view and manage user's specific preferences.
+
+### Bug Fix
+
+This version addressed the following bugs:
+
+* 
```

Comment:
It has been fixed a bug that prevented the correct redirect from the Project Metrics Homepage to the Runtime Area when clicking on the Pods Status of a specific environment.